types/hooks/screens/useChatScreenHook.types.ts

```diff
@@ -8,7 +8,7 @@ interface UseChatScreenHook {
   chats: ChatSchema[];
   goBackFromChatScreen: () => void;
   goToChatInfoScreen: (params?: any) => void;
-  sendChat: (message: string) => Promise<void>;
+  sendChat: (message: string, attachments: any) => Promise<void>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendChat` function now accepts an additional parameter `attachments`. However, the type of `attachments` is defined as `any`, which is not a good practice in TypeScript. It's better to define a specific type for `attachments` to ensure type safety and improve code readability.

```diff
- sendChat: (message: string, attachments: any) => Promise<void>;
+ sendChat: (message: string, attachments: AttachmentType[]) => Promise<void>;
```

In this case, `AttachmentType` could be a new type that you define to represent the structure of an attachment. For example:

```typescript
type AttachmentType = {
  name: string;
  type: string;
  size: number;
  data: Blob;
};
```
This is just an example, you should adjust it according to your actual needs.


<!-- This is an auto-generated comment by OSS CodeRabbit -->